cpp/tests/community/louvain_test.cu

```diff
@@ -123,19 +135,31 @@ TEST(louvain_renumbered, success)
 
   raft::handle_t handle;
 
-  std::tie(num_level, modularity) = cugraph::louvain(handle, G, result_v.data().get());
+  // "FIXME": remove this check once we drop support for Pascal
+  //
+  // Calling louvain on Pascal will throw an exception, we'll check that
+  // this is the behavior while we still support Pascal (device_prop.major < 7)
+  //
+  cudaDeviceProp device_prop;
+  CUDA_CHECK(cudaGetDeviceProperties(&device_prop, 0));
```

Comment:
Again, RAFT has a utility function to get cudaDeviceProp.

Reply:
Updated